.circleci/config.yml

```diff
@@ -170,6 +170,38 @@ jobs:
           root: src/workspace
           paths:
             - "*"
+  build-ceph-16:
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: quay.io/cybozu/ubuntu-dev:20.04
+    resource_class: 2xlarge+
+    steps:
+      - checkout
+      - run:
+          name: Check TAG file
+          command: |
+            ceph_tag_exists="$(./tag_exists ceph ceph-16 -dev)"
+            cephcsi_tag_exists="$(./tag_exists cephcsi)"
```

Comment:
we have no need to build ceph if it is necessary to build cephcsi, can delete this.